native/cocos/scene/ReflectionProbe.h

```diff
@@ -123,10 +134,19 @@ class ReflectionProbe final {
 
     inline bool validate() const { return _cubemap != nullptr; }
 
-private:
+    void initBakedTextures();
+    void captureCubemap();
+
+    inline const ccstd::vector<IntrusivePtr<cc::RenderTexture>>& getBakedCubeTextures() { return _bakedCubeTextures; }
+    void resetCameraParams();
+    void updateCameraDir(int32_t faceIdx);
+    Vec2 renderArea() const;
```

Comment:
Function name should starts with a verb. So it is better to rename `renderArea ` to `getRenderArea `